cloud/blockstore/tools/csi_driver/internal/driver/node_test.go

```diff
@@ -663,6 +656,11 @@ func TestGetVolumeStatCapabilitiesWithoutVmMode(t *testing.T) {
 	targetFsPathPattern := filepath.Join(tempDir,
```

Comment:
This looks like valid path for vmMode. For test 'WithoutVmMode' I believe we need something like "volumeDevices/publish/...".

Reply:
it's a valid pattern for "mount" mode: https://github.com/ydb-platform/nbs/blob/main/cloud/blockstore/tests/csi_driver/test.py#L350

your example for "block" mode

Reply:
Right! My bad